src/lumigo_tracer/parsers/event_parser.py

```diff
@@ -0,0 +1,75 @@
+import re
+from collections import OrderedDict
+from typing import Dict
+
+API_GW_REGEX = re.compile(rf".*execute-api.*amazonaws\.com.*")
+API_GW_KEYS_ORDER = [
+    "resource",
+    "path",
+    "httpMethod",
+    "queryStringParameters",
+    "multiValueQueryStringParameters",
+    "pathParameters",
+    "body",
+    "requestContext",
+    "headers",
+]
+
+API_GW_PREFIX_KEYS_HEADERS_DELETE_KEYS = (
+    "cookie",
+    "X-Amz",
+    "Accept",
+    "CloudFront",
+    "Via",
+    "X-Forwarded",
+    "sec-",
+)
+
+API_GW_REQUEST_CONTEXT_FILTER_KEYS = ["authorizer"]
+
+API_GW_KEYS_DELETE_KEYS = ["multiValueHeaders"]
+
+
+def parse_event(event: Dict):
+    try:
+        if _is_api_gw_event(event):
+            return _parse_api_gw_event(event)
+        else:
+            return event
+    except Exception:
+        return event
+
+
+def _is_api_gw_event(event: Dict) -> bool:
+    return API_GW_REGEX.match(event["requestContext"]["domainName"]) is not None
+
+
+def _parse_api_gw_event(event: Dict):
```

Comment:
As I wrote in the concluence, this function should be dependent on some user input (env var?) to decide whether to cut the body or not....